packages/eslint-config-airbnb/rules/react.js

```diff
@@ -224,10 +224,6 @@ module.exports = {
       skipUndeclared: false
     }],
 
-    // Prevent missing React when using JSX
-    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md
-    'react/react-in-jsx-scope': 'error',
-
```

Comment:
this lint rule *must* be enabled for anyone using react < 16.14, and anyone who's not using the new jsx transform.

Is there a way this can be detected, so we can do the right thing?

Note as well, the `react/jsx-uses-react` rule must also be disabled for this scenario.

Reply:
I see. It's more complex than anticipated in this case. Unfortunately I can't provide a better solution at the moment so I will close the PR. Thanks for the swift review.